### PR TITLE
nc-news-6

### PR DESCRIPTION
**Pull request by 8maziar**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -87,6 +87,43 @@ describe("GET /api/articles", () => {
...
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status 200, should return array of comments for an article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        expect(comments[0].created_at).toBe("2020-11-03T21:00:00.000Z");
```
Comment (Liam310) [inline __tests__/app.test.js:98]: I like the intention here - I think just checking one item might be a little undercooked. I recommend having a look at using [jest-sorted](https://www.npmjs.com/package/jest-sorted) as a tool to help you do this neatly!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -87,6 +87,43 @@ describe("GET /api/articles", () => {
...
+  test("status 400, should return obj with message of bad request", () => {
+    return request(app)
+      .get("/api/articles/ten/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+});
+
```
Comment (Liam310) [inline __tests__/app.test.js:135]: Solid error testing in both cases here!

**Review by Liam310 — CHANGES_REQUESTED** [review]
Hey there 👋 

First thing to note: commit messages. Try to phrase your commit messages like you're filling in the blanks of This commit will ____. Avoid using task numbers/descriptions from the challenges we give you, they will have no context for any future prospective employers reading your commit history. A good example might be "add a fetch article by ID endpoint".

There's a ton of great code in here, well done 👏 

Just one thing I'm going to pull you up on - what would happen if I tried to make a request for the comments on an article that had no comments? Currently I think your logic would respond with a 404 not found but... what if we could find the article, it just didn't have any comments? Think about yesterday's lecture and consult the notes if you need an assist.

**Review by RichardNeat — APPROVED** [review]
Nice work implementing that change! All good to merge :)

**Pull request merged**